src/hiveboard_bridge/include/hiveboard_bridge/ReceiveThreadAction.h

```diff
@@ -0,0 +1,24 @@
+#include "IMessageHandler.h"
+#include <hivemind-host/IHiveMindHostDeserializer.h>
+
+#ifndef HIVEBOARD_BRIDGE_RECEIVETHREADACTION_H
+#define HIVEBOARD_BRIDGE_RECEIVETHREADACTION_H
+
+/**
+ * @brief A class that contains all the actions that must be done in a receive thread.
+ */
+class ReceiveThreadAction {
```

Comment:
There is no concept of a thread nowhere in this code or it's implementation, is that normal?

Reply:
Renamed to `ReceiveAction`, since this class only describes what to do when receiving a message.